### DISCUSSION TITLE
Which xml file describes electricity production?

### DISCUSSION THREAD

**Discussion by stm19950929** [JGCRI/gcam-core#204] [Q&A] [answered]
In the configuration, the electricity.xml file is annotated by default. And I want to knowIn which file are the parameters describing electricity production and conversion?

**Comment by mbins** [comment, marked as answer]
`electricity_water.xml` contains the information you want.  It sets up the model structure for electricity generation (competition between fuels and between generation technologies), as well as competition between different cooling systems for thermal power plants.

**Comment by stm19950929** [comment]
> `electricity_water.xml` contains the information you want. It sets up the model structure for electricity generation (competition between fuels and between generation technologies), as well as competition between different cooling systems for thermal power plants.

I've found this file, so thanks^.^

**Comment by github-wbx** [comment]
> `electricity_water.xml` contains the information you want. It sets up the model structure for electricity generation (competition between fuels and between generation technologies), as well as competition between different cooling systems for thermal power plants.
Hi, if I want to change the structure of electricity generation in a specific region,  how to implement it?
